projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-vc.ts

```diff
@@ -4,6 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { PRODUCT_NO_PRICING } from '../sample-data/b2b-bulk-pricing';
```

Comment:
I get the following notification: 'PRODUCT_NO_PRICING' is declared but its value is never read.
Please, remove unused import.